contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/checkstyle.py

```diff
@@ -94,11 +90,16 @@ def checker_pex(self, interpreter):
     else:
       checker_id = hash_all([self._CHECKER_REQ])
 
+    # TODO(CMLivingston): We should be able to build a multi interpreter pex here and avoid
```

Comment:
Stale comment.

Reply:
Thanks,  this is actually relevant and I am checking with John that it is OK to punt this for a later PR before making a ticket for it.